anitya/default_config.py

```diff
@@ -18,7 +18,8 @@
 SECRET_KEY = 'changeme please'
 
 # URL to the database
-DB_URL = 'sqlite:////var/tmp/anitya-dev.sqlite'
+repo_root = path.abspath(path.dirname(path.dirname(__file__)))
+DB_URL = 'sqlite:///{0}/anitya.db'.format(repo_root)
```

Comment:
The main reason was that if this wasn't overridden, it could cause potential permission issues such as Apache not being able to access the database. I just thought it was a friendlier default value.